pysid/io/csv_data.py

```diff
@@ -60,14 +60,15 @@ def load_data(filename, delim=",", skip_rows=1):
     Returns
     -------
     data : ndarray
-    Loaded dataset.
+        Loaded dataset.
     """
     try:
         data = loadtxt(filename, dtype=float, delimiter=delim, skiprows=skip_rows)
         print("csv_data.py::load_data -- Successfully loaded " + filename + ".")
         return data
     except:
```

Comment:
You should replace your except block with:
` ``
except IOError:
        raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), filename) from None
```